src/airflow/dags/common_airflow.py

```diff
@@ -59,6 +60,24 @@
 }
 
 
+# Test if release folder exists:
```

Comment:
```suggestion
```

Reply:
Unfortunately we are still not there yet. (that's why I haven't ask for a re-reivew) For some reason the the gcshook.exists() stopped working. Now I'm trying to find out what is the problem. Keep you posted.

Reply:
I see, thanks for testing it.